<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>von-Neumann-Emulator</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>31</y>
     <width>853</width>
     <height>627</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5" >
    <item>
     <layout class="QHBoxLayout" name="mainLayout" >
      <item>
       <layout class="QVBoxLayout" name="configurationLayout" >
        <property name="sizeConstraint" >
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="storageLayout" >
          <property name="sizeConstraint" >
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="indexregisterLayout" >
            <item>
             <widget class="QLabel" name="labelIndexregister" >
              <property name="text" >
               <string>&lt;b>Indexregister&lt;/b></string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidgetIndexregister" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="dataLayout" >
            <item>
             <widget class="QLabel" name="labelData" >
              <property name="text" >
               <string>&lt;b>Data&lt;b></string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidgetData" />
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="ioLayout" >
          <item>
           <layout class="QVBoxLayout" name="inputstatusLayout" >
            <item>
             <layout class="QVBoxLayout" name="inputLayout" >
              <item>
               <widget class="QLabel" name="labelInput" >
                <property name="font" >
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text" >
                 <string>&lt;b>Eingabe&lt;/b></string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="InputCrtlLayout" >
                <item>
                 <widget class="QLineEdit" name="lineEditInput" />
                </item>
                <item>
                 <widget class="QToolButton" name="toolBtnOpenInput" >
                  <property name="toolTip" >
                   <string>Eingabeband-Datei Ã¶ffnen</string>
                  </property>
                  <property name="icon" >
                   <iconset resource="../res/icons.qrc" >
                    <normaloff>:/famfamfam/page_add.png</normaloff>:/famfamfam/page_add.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="toolBtnClearInput" >
                  <property name="toolTip" >
                   <string>Eingabeband leeren</string>
                  </property>
                  <property name="icon" >
                   <iconset resource="../res/icons.qrc" >
                    <normaloff>:/famfamfam/page_delete.png</normaloff>:/famfamfam/page_delete.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="listWidgetInput" />
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="statusLayout" >
              <item row="0" column="0" >
               <widget class="QLabel" name="labelOperation" >
                <property name="text" >
                 <string>&lt;p align="right">&lt;b>Operation:&lt;/b>&lt;/p></string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" >
               <widget class="QLabel" name="lblOperation" >
                <property name="text" >
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="2" >
               <spacer name="horizontalSpacerOperation" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>95</width>
                  <height>17</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item rowspan="2" row="1" column="1" >
               <widget class="QLabel" name="lblAC" >
                <property name="text" >
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="2" >
               <spacer name="horizontalSpacerAC" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>95</width>
                  <height>17</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="0" >
               <widget class="QLabel" name="labelPC" >
                <property name="text" >
                 <string>&lt;p align="right">&lt;b>PC:&lt;/b>&lt;/p></string>
                </property>
               </widget>
              </item>
              <item row="3" column="1" >
               <widget class="QLabel" name="lblPC" >
                <property name="text" >
                 <string/>
                </property>
               </widget>
              </item>
              <item rowspan="2" row="2" column="2" >
               <spacer name="horizontalSpacerPC" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>95</width>
                  <height>21</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="4" column="0" >
               <widget class="QLabel" name="labelSR" >
                <property name="text" >
                 <string>&lt;p align="right">&lt;b>SR:&lt;/b>&lt;/p></string>
                </property>
               </widget>
              </item>
              <item row="4" column="1" >
               <widget class="QLabel" name="lblSR" >
                <property name="text" >
                 <string/>
                </property>
               </widget>
              </item>
              <item row="4" column="2" >
               <spacer name="horizontalSpacerSR" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>95</width>
                  <height>17</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item rowspan="2" row="1" column="0" >
               <widget class="QLabel" name="labelAC" >
                <property name="text" >
                 <string>&lt;p align="right">&lt;b>AC:&lt;/b>&lt;/p></string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="outputLayout" >
            <item>
             <widget class="QLabel" name="labelOutput" >
              <property name="text" >
               <string>&lt;b>Ausgabeband&lt;/b></string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidgetOutput" />
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="controlLayout" >
          <item>
           <spacer name="horizontalSpacerControlLeft" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolBtnPlay" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="icon" >
             <iconset resource="../res/icons.qrc" >
              <normaloff>:/famfamfam/control_play_blue.png</normaloff>:/famfamfam/control_play_blue.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolBtnNext" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="icon" >
             <iconset resource="../res/icons.qrc" >
              <normaloff>:/famfamfam/control_end_blue.png</normaloff>:/famfamfam/control_end_blue.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolBtnPause" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="icon" >
             <iconset resource="../res/icons.qrc" >
              <normaloff>:/famfamfam/control_pause_blue.png</normaloff>:/famfamfam/control_pause_blue.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolBtnStop" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="icon" >
             <iconset resource="../res/icons.qrc" >
              <normaloff>:/famfamfam/control_stop_blue.png</normaloff>:/famfamfam/control_stop_blue.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacerControlRight" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolBtnNumber" >
            <property name="toolTip" >
             <string>Quelltext nummerieren</string>
            </property>
            <property name="icon" >
             <iconset resource="../res/icons.qrc" >
              <normaloff>:/famfamfam/folder_go.png</normaloff>:/famfamfam/folder_go.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="sourcecodeLayout" >
        <item>
         <widget class="QLabel" name="labelSourcecode" >
          <property name="text" >
           <string>&lt;b>Quellcode&lt;/b></string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txtEditSourcecode" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="font" >
           <font>
            <family>DejaVu Sans Mono, Courier New</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;Datei</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="separator" />
    <addaction name="actionSave" />
    <addaction name="actionSaveAs" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Hilfe</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAboutQt" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>658</y>
     <width>853</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <action name="actionNew" >
   <property name="icon" >
    <iconset resource="../res/icons.qrc" >
     <normaloff>:/tango/document-new.png</normaloff>:/tango/document-new.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Neu</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset resource="../res/icons.qrc" >
     <normaloff>:/tango/document-open.png</normaloff>:/tango/document-open.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Ãffnen...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset resource="../res/icons.qrc" >
     <normaloff>:/tango/document-save.png</normaloff>:/tango/document-save.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Speichern</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="icon" >
    <iconset resource="../res/icons.qrc" >
     <normaloff>:/tango/document-save-as.png</normaloff>:/tango/document-save-as.png</iconset>
   </property>
   <property name="text" >
    <string>Speichern &amp;unter...</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="../res/icons.qrc" >
     <normaloff>:/tango/system-log-out.png</normaloff>:/tango/system-log-out.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Beenden</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>Ã&amp;ber...</string>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="text" >
    <string>Ãber &amp;Qt...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>listWidgetIndexregister</tabstop>
  <tabstop>listWidgetData</tabstop>
  <tabstop>lineEditInput</tabstop>
  <tabstop>listWidgetInput</tabstop>
  <tabstop>listWidgetOutput</tabstop>
  <tabstop>txtEditSourcecode</tabstop>
  <tabstop>toolBtnPlay</tabstop>
  <tabstop>toolBtnNext</tabstop>
  <tabstop>toolBtnPause</tabstop>
  <tabstop>toolBtnStop</tabstop>
 </tabstops>
 <resources>
  <include location="../res/icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>231</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
